python/lsst/pipe/tasks/computeExposureSummaryStats.py

```diff
@@ -876,33 +876,36 @@ def compute_ap_corr_sigma_scaled_delta(
 
 
 def compute_magnitude_limit(
-        psfSigma,
+        psfArea,
         skyBg,
         zeroPoint,
         readNoise=0,
+        gain=1.0,
         snr=5
 ):
     """Compute the expected point-source magnitude limit at a given
     signal-to-noise ratio given the exposure-level metadata. Based on
     the signal-to-noise formula provided in SMTN-002 (see LSE-40 for
     more details on the calculation).
 
-    SNR = C / sqrt(C + (B + sigma_inst**2) * neff )
+    SNR = C / sqrt( C/g + (B/g + sigma_inst**2) * neff )
```

Comment:
Could you add some of the info (variable definitions, etc.) that you included in your Slack post?

> "...where C = source counts, B = sky background counts, sigma_inst = instrumental noise (dominated by readout noise), neff = the effective number of pixels within the source footprint, and g = the gain. Including the instrumental noise means that this metric should be valid for observations in regimes that are sky-noise-dominated (r, i, z, y), read-noise-dominated (u), and in between (g)."